workspace/workspace_queries.go

```diff
@@ -182,6 +183,10 @@ func (w *Workspace) getQueryFromFile(filename string) (string, bool, error) {
 
 // does this resource name look like a control or query
 func isNamedQueryOrControl(name string) bool {
+	openBracketIdx := strings.Index(name, "(")
```

Comment:
You could probably make a case for adding this logic in [`ParseResourceName`](https://github.com/turbot/steampipe/blob/main/steampipeconfig/modconfig/parsed_name.go#L14-L39), but I opted to make a more localized fixed given how broadly `ParseResourceName` is used throughout the codebase.

Reply:
agreed, that makes sense for now

Reply:
I would either add a short comment here explaining why it it necessary